site/content/docs/main/file-system-backup.md

```diff
@@ -255,7 +268,12 @@ Instructions to back up using this approach are as follows:
 
 Velero, by default, uses this approach to discover pod volumes that need to be backed up using FSB. Every pod 
 containing a volume to be backed up using FSB must be annotated with the volume's name using the 
-`backup.velero.io/backup-volumes` annotation.  
+`backup.velero.io/backup-volumes` annotation.
+
+**Note:** Volumes not annotated for FSB will be considered for volume snapshots if:
+- `--snapshot-volumes` is not set to `false`
+- A VolumeSnapshotLocation is configured
+- The volume type supports snapshots  
```

Comment:
- the volume supports snapshots (either CSI or native)
- either the volume is a CSI volume and CSI snapshots are enabled or there is a compatible VolumeSnapshotLocation configured